package/Dockerfile.suc

```diff
@@ -3,8 +3,10 @@ FROM ${ALPINE} as kubectl
 RUN apk add -U curl
 RUN ARCH=$(uname -m) && \
     if [ "$ARCH" = "x86_64" ]; then ARCH=amd64; fi && \
-    curl -L -o /usr/bin/kubectl https://dl.k8s.io/release/v1.24.0/bin/linux/${ARCH}/kubectl && \
+    if [ "$ARCH" = "aarch64" ]; then ARCH=arm64; fi && \
+    curl -L -f -o /usr/bin/kubectl https://dl.k8s.io/release/v1.24.0/bin/linux/${ARCH}/kubectl && \
     chmod +x /usr/bin/kubectl
+RUN /usr/bin/kubectl version --client
```

Comment:
Do we need this `RUN` step to check if the binary is working, or is this debug logging?

Reply:
It makes sure that the binary actually works for the arch it's being built on. It could catch bad files, downloading the wrong arch, etc.

Reply:
@sbstp , thank you for thinking about it and adding this "defensive" check here. Though since it's not exactly trivial why we do that, a comment would be nice in cases like this. (No need to go back and add it though)